modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -1575,7 +1593,10 @@ impl Wallet {
                             let _ = fee_rate_tx.send(feerate_with_multiplier);
                         }
                         Ok(None) => {
-                            debug!(target: LOG_MODULE_WALLET, "Bitcoin node did not return a fee rate");
+                            // Regtest node never returns fee rate, so no point spamming about it
+                            if !is_running_in_test_env() {
+                                warn!(target: LOG_MODULE_WALLET, "Bitcoin node did not return a fee rate");
+                            }
```

Comment:
Wouldn't that also be quite spammy on a newly started federation with a fresh bitcoind? Doesn't it take a moment to start returning a fee rate?

Reply:
`bitcoind` should have been running for some time syncing up, no?

Also it's not that spammy. Just annoying when debugging having to scroll through tens of screens worth of it.